src/Reveal.Sdk.Dom.Tests/Visualizations/CandleStickVisualizationFixture.cs

```diff
@@ -0,0 +1,354 @@
+                "AdHocExpandedElements" : [ ],
+                "Rows" : [ {
+                  "_type" : "DimensionColumnSpecType",
+                  "SummarizationField" : {
+                    "_type" : "SummarizationDateFieldType",
+                    "DateAggregationType" : "Day",
+                    "DrillDownElements" : [ ],
+                    "ExpandedItems" : [ ],
+                    "FieldName" : "Date"
+                  }
+                } ]
+              }
+            } ]
+            """;
+
+        //Act
+        var document = new RdashDocument("My Dashboard");
+
+        var excelDataSourceItem = new RestDataSourceItem("Marketing Sheet")
+        {
+            Id = "080cc17d-4a0a-4837-aa3f-ef2571ea443a",
+            Subtitle = "Excel Data Source Item",
+            Url = "http://dl.infragistics.com/reportplus/reveal/samples/Samples.xlsx",
+            IsAnonymous = true,
+            ResourceItem = new DataSourceItem
+            {
+                Id = "d593dd79-7161-4929-afc9-c26393f5b650",
+                DataSourceId = "33077d1e-19c5-44fe-b981-6765af3156a6",
+                Title = "Marketing Sheet",
+                Subtitle = "Excel Data Source Item",
+                HasTabularData = true,
+                HasAsset = false,
+                Properties = new Dictionary<string, object>
+                {
+                    { "Url", "http://dl.infragistics.com/reportplus/reveal/samples/Samples.xlsx" }
+                }
+            },
+            Fields = new List<IField>
+            {
+                new NumberField("Open"),
+                new NumberField("High"),
+                new NumberField("Low"),
+                new NumberField("Close")
+            }
+        };
+        excelDataSourceItem.UseExcel("Marketing");
+
+        document.Visualizations.Add(new CandleStickVisualization("Candle Stick Visualization", excelDataSourceItem)
+            {
+                Id = "60344a4a-d0ce-4364-9f8c-acfbb8caa32e",
+                IsTitleVisible = true,
+                Description = "Create Candle Stick Visualization"
+            }
+            .SetLabel(new DateDataField("Date") { AggregationType = DateAggregationType.Day })
+            .SetOpen("Open")
+            .SetHigh("High")
+            .SetLow("Low")
+            .SetClose("Close")
+            .ConfigureSettings(settings =>
+            {
+                settings.SchemaTypeName = "SampleSchema";
+            }));
+
+        document.Filters.Add(new DashboardDataFilter("Spend", excelDataSourceItem));
+        document.Filters.Add(new DashboardDateFilter("My Date Filter"));
+
+        RdashSerializer.SerializeObject(document);
+        var json = document.ToJsonString();
+        var actualJson = JObject.Parse(json)["Widgets"];
+        var expected = JArray.Parse(expectedJson);
+
+        var expectedStr = JsonConvert.SerializeObject(expected);
+        var actualStr = JsonConvert.SerializeObject(actualJson);
```

Comment:
Quite complex if we have expected json -> Parse it -> Convert it to json again. Check it again